### PR TITLE
[libra framework] cleanup LibraAccount and use withdraw everywhere

### PR DESCRIPTION
**Pull request by emmazzz**
## Motivation

This PR does one pass of cleaning up specs and documentation for `LibraAccount`. It also makes the naming consistent by using "withdraw" instead of "withdrawal" to describe capability everywhere.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### CODE REVIEW

File: language/stdlib/modules/LibraAccount.move [hunk 1]
```diff
@@ -186,7 +184,6 @@ module LibraAccount {
     /// important to the semantics of the system. Those codes also need to be
     /// directly used in aborts instead of augmenting them with a category
     /// via the `Errors` module.
-    /// const PROLOGUE_EACCOUNT_FROZEN: u64 = 1000;
```
Comment (emmazzz) [inline language/stdlib/modules/LibraAccount.move:189]: This was quite subtle and hard to see.

File: language/stdlib/modules/LibraAccount.move [hunk 2]
```diff
@@ -826,9 +837,8 @@ module LibraAccount {
         ensures spec_holds_own_key_rotation_cap(cap.account_address);
     }
 
-
     /// Add balances for `Token` to `new_account`.  If `add_all_currencies` is true,
-    /// then add for each of the three token types.
+    /// then add for both token types.
```
Comment (emmazzz) [inline language/stdlib/modules/LibraAccount.move:841]: Since we only have two tokens now.

File: language/stdlib/modules/LibraAccount.move [hunk 3]
```diff
@@ -886,8 +896,6 @@ module LibraAccount {
         new_account: signer,
         auth_key_prefix: vector<u8>,
     ) acquires AccountOperationsCapability {
-        // Magically create a signer so we can do `move_to` to store LibraAccount, etc.
-        // at the new address.
```
Comment (emmazzz) [inline language/stdlib/modules/LibraAccount.move:890]: Removed this comment because the following line isn't creating a signer.

**Review by wrwg — APPROVED** [review]

**Pull request closed without merging**